golangci-lint.sh

```diff
@@ -0,0 +1,11 @@
+
+#!/bin/bash
+
+cd api
```

Comment:
Will this work if you run commit from a directory that is not the top level directory?

or do we need something like the following instead:

```
PROJECT_ROOT=$(git rev-parse --show-toplevel)
cd $PROJECT_ROOT/api
```

Reply:
It works if committed from any sub dir also
![image](https://user-images.githubusercontent.com/91852476/213296218-5aceb477-3553-4c8a-baa8-a18f2ae5ecde.png)